mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -235,6 +235,7 @@ class MultilayerPerceptronClassifier @Since("1.5.0") (
    */
   override protected def train(dataset: Dataset[_]): MultilayerPerceptronClassificationModel = {
     val myLayers = $(layers)
+    require(myLayers != null, "layers parameter not set.")
```

Comment:
is this really required? because `$layers` should call `getOrDefault` and if there is no default and it is not set, it already throws an erorr